cluster/manifests/aws-load-balancer-controller/deployment.yaml

```diff
@@ -0,0 +1,78 @@
+# {{- if and (eq .Cluster.Provider "zalando-eks") (eq .Cluster.ConfigItems.eks_ip_family "ipv6") }}
+apiVersion: apps/v1
```

Comment:
Will this collide with k8s-ingress-on-aws controller? Will we create 2 LBs for the same ingress?

Reply:
No, this controller is configured to only process `Service` with `Type: LoadBalancer`.

It can do `Ingress` but since we have our own controller for that, it's disabled via `--feature-gates=ServiceTypeLoadBalancerOnly=true`

Reply:
Thanks for the clarification 🙏🏻!